src/parser/line.rs

```diff
@@ -118,15 +119,15 @@ impl<'l> Line<'l> {
 
 impl<'l> From<&'l str> for Line<'l> {
     fn from(line: &'l str) -> Self {
-        if line.is_empty() || line.starts_with(HASH) {
+        if line.is_empty() || line.starts_with(HASH) || line.eq(EXPORT) {
```

Comment:
```suggestion
        if line.is_empty() || line.starts_with(HASH) || line.trim().eq(EXPORT) {
```

Reply:
I think `line.eq(EXPORT)` check would fail if the line has `export` with trailing + leading spaces

Reply:
Good catch! I did not think of this.

Weird thing, however, is that -me doing TDD- I wrote the test first, and found it passed. Turns out the match() below falls back into a catchall that will return a Line::empty already, since we don't match any valid KEY Pattern.

I went ahead and added the trim() nonetheless, as it will allow early exit in cases of empty-spaced-export. But I guess that could count as premature optimization. Please let me know if you still want to keep it in. I kept one test to ensure any future refactorings will still handle the parsing as designed here.

Reply:
Ahhhh, I missed that. I guess we don't need trimming.